tools/info/main.cpp

```diff
@@ -620,7 +619,10 @@ int main(int argc, const char** argv)
 
             GFXRECON_WRITE_CONSOLE("\nDevice memory allocation info:");
             GFXRECON_WRITE_CONSOLE("\tTotal allocations: %" PRIu64, stats_consumer.GetAllocationCount());
-            GFXRECON_WRITE_CONSOLE("\tMin allocation size: %" PRIu64, stats_consumer.GetMinAllocationSize());
+            GFXRECON_WRITE_CONSOLE("\tMin allocation size: %" PRIu64,
+                                   (stats_consumer.GetMinAllocationSize() == std::numeric_limits<uint64_t>::max()
+                                        ? 0
+                                        : stats_consumer.GetMinAllocationSize()));
```

Comment:
When will `GetMinAllocationsSize()` return the same as `max()`?

Comments here to explain the unexpected code would be helpful.

Reply:
https://github.com/LunarG/gfxreconstruct/blob/dev/tools/info/main.cpp#L494

Reply:
`min_allocation_size_`  initialize `std::numeric_limits<uint64_t>::max()`, so if there is no memory allocation, GetMinAllocationsSize() would be the same as max().

Reply:
I think Bob is suggesting adding a comment above that line like:
```
// If there were no allocations, report zero rather than the default value:
GFXRECON_WRITE_CONSOLE("\tMin allocation size: %" PRIu64, ...
```
Best practice would be to define a constant somewhere and have the initialisation and this test both use it so the two places can't get out of sync and questions like this are answerable by grepping the name of the constant:
```
constexpr uint64_t GFXR_MIN_ALLOC_INITIAL = std::numeric_limits<uint64_t>::max();
```

Reply:
Thank you. Added.

Reply:
Could we just not output `Min allocation size` and `Max allocation size` if there were no allocations?

Reply:
@bradgrantham-lunarg Good idea! It won't print min and max allocation size when allocation count is 0.